uk.co.compucorp.civicrm.tasksassignments/js/test/tasks-assignments/controllers/modal/modal-assignment.controller.spec.js

```diff
@@ -410,7 +435,11 @@ define([
       });
 
       describe('after selecting a default assignee', function () {
-        var contact = { id: _.uniqueId(), display_name: 'Jon Snow' };
+        var contact = {
+          contact_id: _.uniqueId(),
+          display_name: 'Jon Snow',
```

Comment:
I personally prefer Aria, but OK, let it be John. But could we also move this to a var somewhere in the top of the file? In addition to a sample date we may even have an object:

```js
var sampleData = {
  date: '2013-01-01',
  contactName: 'John Snow'
};
```